### ISSUE TITLE
NVDA fails to see text in the status bar of Mozilla Thunderbird 68 and newer

### ISSUE THREAD

**Issue by k-kolev1985** [nvaccess/nvda#10460]
### Steps to reproduce:
1. Install Mozilla Thunderbird 68 and configure it with at least one e-mail or RSS account.
2. In Mozilla Thunderbird, go to menu bar -> View -> Toolbars and if "Status bar" is not checked, press ENTER to check it so it shows on the bottom of the main Mozilla Thunderbird window.
3. In Mozilla Thunderbird, go with F6 to the tree view with folders and select one folder, so information for it is displayed in the status bar.
4. Execute NVDA's command to read the status bar (e.g. NVDA+End).

### Actual behavior:
NVDA reports that there is no text in the status bar, even though there is some text displayed there.

### Expected behavior:
NVDA should report the text in the status bar, as it did in versions of Mozilla Thunderbird prior to version 68.

### System configuration
- Operating system: Windows 10 Pro version 1903 (build 18362.449), 64-bit, in Bulgarian with all locale settings set to "Bulgarian".
- NVDA version: alpha-19065,1b7cddfd, in bulgarian.
- Mozilla Thunderbird version: 68.2.1, 32-bit, in bulgarian.
- Processor: Intel Core i5-4670 at 3.40GHz.
- RAM Memory: 8.00GB.
- Graphics: MSI GeForce GTX-1050TI Gaming X 4G, 4096MB dedicated memory + 4065MB shared system memory, desktop resolution set to 1920x1080.
- Sound Card: Realtek ALC887 at Intel Lynx Point PCH - High Definition Audio Controller.

#### Does the issue still occur after restarting your PC?
Yes.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes - 2019.2.1. The Behavior is the same.

**Comment by amirsol81** [comment]
This has been resolved in Thunderbird version 68.7.0. However, the new issue is that NVDA now tends to automatically read whatever Thunderbird displays in its status bar, thereby making the screen reader quite verbose. I suggest that the auto-reading of the status bar in Thunderbird version 68.7.0 and newer be suppressed. This is with NVDA Version alpha-19925,2d73010e.

**Comment by Adriani90** [comment]
this is being investigated by the TB team apparently, closing this in favor of #10975.
